ansible_collections/arista/avd/plugins/README.md

```diff
@@ -204,6 +210,142 @@ To use this filter:
 
 !!! Note this is not using the same range syntax as EOS for modular or break-out ports. On EOS `et1/1-2/4` gives you `et1/1, et1/2, et1/3, et1/4, et2/1, et2/2, et2/3, et2/4` on a fixed switch, but a different result on a modular switch depending on the module types. In AVD the same range would be `et1-2/1-4`
 
+### IP Address filters
+
+Four filters are currently supported to handle all of the IP Address related calculations.
+
+#### ipaddr filter
+
+The ```arista.avd.ipaddr``` filter provides the capabilities to calculate all IP address related functionality like, `subnet`, `address`, `subnet size`, `network ip` etc. The input can be an IP address or a list of IP addresses and the output will be of similar data-type.
+
+This filter supports both `IPv4` and `IPv6` version types. If no IP version is specified, then the filter calculates for both of the versions. This filter takes two arguments apart from the IP address(es) as input, which are `method` and `version`. The supported versions are `4`, and `6` (integers), which represent the **ipv4** and **ipv6** respectively. The supported `methods` are:
+
+1) `validate`: Returns the valid IP addresses or Network IPs based on the version type. If no version number is provided then both IPv4 and IPv6 IPs are returned based on the input data-type(string or list). This is the default method in the `ipaddr` filter.
+
+    ```example
+    ip_pool = ['192.168.0.1', '192.168.0', 'fe08::1', '192.168.1.0/24', '', True, 'test12.com']
+    {{ ip_pool | arista.avd.ipaddr }}
+    Output: ['192.168.0.1', 'fe08::1', '192.168.1.0/24']
+    ```
+
+    The below method is same as above.
+
+    ```example
+    ip_pool = ['192.168.0.1', '192.168.0', 'fe08::1', '192.168.1.0/24', '', True, 'test12.com']
+    {{ ip_pool | arista.avd.ipaddr('validate') }}
+    Output: ['192.168.0.1', 'fe08::1', '192.168.1.0/24']
+    ```
+
+1) `address`: Returns the valid IP address(es) only.
+
+    ```example
+    ip_pool = ['192.168.0.1', '192.168.0', 'fe08::1', '192.168.1.0/24', '', True]
+    {{ ip_pool | arista.avd.ipaddr('address') }}
+    Output: ['192.168.0.1', 'fe08::1']
+    ```
+
+1) `net`: Returns only the Network IPs.
```

Comment:
Does it work in mkdocs?

Reply:
Just tested it and seems to work as well in mkdocs (though the expected syntax is `1.` rather than `1)` and the Github syntax also recommends using `1.` syntax: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#lists